view/frontend/templates/boltaccount.phtml

```diff
@@ -51,7 +51,7 @@ $checkoutKey = $block->getCheckoutKey();
                 // When magento constructs mobile menu it copies elements instead of moving
                 // We need to wait for this coping, remove first bolt-account-login div
                 // And only ofter that insert account.js script
-                let timerId = setTimeout(function boltAccountLookup() {
+                var timerId = setTimeout(function boltAccountLookup() {
```

Comment:
This seems unrelated?

Reply:
nvm missed bit in description

Reply:
@njfaries `let` is a keyword present in es6, which is not supported in es5 for defining variables. Unsure if it causes any problems with IE, but the change is required in order to fulfill the pre-commit hook.

Reply:
This shouldn't be an issue regardless, since `timerId` is only defined in the block and not called globally.